File: Knowledge/TradeX-MCP/nhsv-admin/src/main/java/com/difisoft/nhsv/admin/domain/request/CopyTradingOrderRequest.java
package com.difisoft.nhsv.admin.domain.request;

import lombok.Data;

import javax.validation.constraints.NotBlank;
import java.time.LocalDateTime;

@Data
public class CopyTradingOrderRequest {
    private Long copyPortfolioId;
    @NotBlank
    private Long subScriberId;
    private LocalDateTime fromDate;
    private LocalDateTime toDate;
    private String stockCode;
    private String sellBuyType;
    private int page = 0;
    private int size = 20;
}

File: Knowledge/TradeX-MCP/nhsv-admin/src/main/webapp/app/custom/model/copy-trading-order.request.ts
export class CopyTradingOrderRequest {
  copyPortfolioId?: number;
  subScriberId?: number;
  fromDate?: Date;
  toDate?: Date;
  stockCode?: string;
  sellBuyType?: string;
  page?: number = 0;
  size?: number = 10;
}
